apps/dot_ext/models.py

```diff
@@ -76,14 +78,12 @@ def get_allowed_schemes(self):
             allowed_schemes.append(scheme)
         return allowed_schemes
 
-    def save(self, commit=True, **kwargs):
-        if commit:
-            # Write the TOS that the app developer agreed to.
-            self.op_tos_uri = settings.TOS_URI
-            super(Application, self).save(**kwargs)
-            logmsg = "%s agreed to %s for the application %s on %s" % (self.user, self.op_tos_uri,
-                                                                       self.name, self.updated)
-            logger.info(logmsg)
+    def set_tos_agree(self):
```

Comment:
find a way to hijack or default setting or getting op_tos_uri since it looks like it is always set to TOS_URI. write tests to show no change in behavior

Reply:
I'm thinking the purpose of updating the op_tos_uri is whenever the developer agree's to the ToS via the checkbox on the edit or create application forms. 
Maybe setting via the clean_agree() via the CustomRegisterApplicationForm is a good place to set it?